libs/e2e-mock/client/src/lib/e2e-mock-client.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { AddKompakt, AddKompaktResponse } from "e2e-mock-server"
+import clientEmiter from "./ipc-client/ipc-client"
+
+export const E2EMockClient = {
+  checkConnection: () => {
```

Comment:
🟢 
Perhaps it would be worth considering renaming the checkConnection method to isConnected to better reflect that it returns a boolean value?

Reply:
I understand the idea but this is a function so it should contain a verb in the name